third_party/warp_transducer/CMakeLists.txt

```diff
@@ -0,0 +1,142 @@
+# Modified from HawkAaron/warp-transducer/CMakeLists.txt to build statically
+
+IF (APPLE)
+    cmake_minimum_required(VERSION 3.4)
+ELSE()
+    cmake_minimum_required(VERSION 2.8)
+ENDIF()
+
+project(rnnt_release)
+
+IF (NOT APPLE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
+ENDIF()
+
+IF (APPLE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O2")
```

Comment:
This looks conflict with the `'-std=c++14'` flag given in `build_tools/setup_helpers/extension.py`